meson.build

```diff
@@ -230,7 +221,7 @@ if get_option('pkgconfig')
     libraries : librc,
     subdirs : ['openrc'],
     variables: {
-      'rc_path': rc_libexecdir,
+      'pluginsdir': get_option('prefix') / get_option('libdir') / 'rc/plugins',
```

Comment:
we already `libdir = get_option('libdir')` above, we probably should reuse it instead of calling get_option many times